kotlin-code-style.md

```diff
@@ -7,17 +7,6 @@ the [Kotlin documentation].
 
 # Classes and Inheritance
 
-## Constructors
-
-As often as possible we should specify if the constructor properties are mutable
-(`var`) or read-only (`val`):
-
-```kotlin
-class Person(val firstName: String, val lastName: String, var age: Int) {
```

Comment:
The code in here is good, it was the way it was formulated that was midleading. Not sure why it's marked as removed here, since it's kept in a58b6267f4599399571235b1d088bb5ced5a4920. :stuck_out_tongue: